src/types/globals/Window.js

```diff
@@ -7,7 +7,7 @@ import type { IDBFactory, IDBKeyRange } from '../indexeddb';
 import type { SymbolTableAsTuple } from '../../profile-logic/symbol-store-db';
 import type { GoogleAnalytics } from '../../utils/analytics';
 
-declare type GeckoProfiler = {
+declare type $GeckoProfiler = {
```

Comment:
Changed it so that we don't confuse it with a "real" global type.

Reply:
The `$` sign is used for some types in the Flow world; the one I thought of when doing this change is [`$TypedArray`](https://github.com/facebook/flow/blob/c7e0cb5c1fb8e0b125e17581c43ddfcf7d5b58b9/lib/core.js#L668): it's not an existing DOM object but it's useful to have it as a type (in this case, this represents any `XXXArray` type).

In this case, I thought that having a global type GeckoProfiler isn't a good idea.
But maybe I can define it in a type, and import it here, like we do with `GoogleAnalytics`?

Reply:
I tried to do the last line, and we lose the coverage... I think that when we import external types in these files that are loaded first (the ones defined in `.flowconfig`) they just don't work. We do that for others as we can see, and they don't work either, so I believe we should likely do the same for these other types, which is a bit painful.